apps/bfd-server/bfd-server-launcher/src/main/assembly/bin/bfd-server-launcher.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Same as the other shell script. Good idea to run these through shellcheck (`brew install shellcheck && shellcheck apps/bfd-server/bfd-server-launcher/src/main/assembly/bin/bfd-server-launcher.sh`)